obspy/signal/tests/test_spectral_estimation.py

```diff
@@ -205,6 +205,77 @@ def test_PPSD(self):
             np.testing.assert_array_equal(ppsd_loaded.period_bins,
                                           binning['period_bins'])
 
+    def test_PPSD_w_IRIS(self):
+        # Bands to be used this is the upper and lower frequency band pairs
+        fresup = [0.2, 0.1]
+        freslow = [0.1, 0.05]
+
+        file_dataANMO = os.path.join(
+            PATH, 'IUANMO.seed')
+        # Read in ANMO data for one day
+        st = read(file_dataANMO)
+
+        # Use a canned ANMO response which will stay static
+        paz = {'gain': 86298.5, 'zeros': [0, 0],
+               'poles': [-59.4313, -22.7121 + 27.1065j, -22.7121 + 27.1065j,
+                         -0.0048004, -0.073199], 'sensitivity': 3.3554*10**9}
+
+        # Make an empty PPSD and add the data
+        ppsd = PPSD(st[0].stats, paz)
+        ppsd.add(st)
+
+        # Get the 50th percentile from the PPSD
+        (per, perval) = ppsd.get_percentile(percentile=50)
+
+        # For each frequency pair we want to compare the mean of the bands
+        for idxfre, curfreUp in enumerate(fresup):
+            perGoodOBSPY = []
+            pervalGoodOBSPY = []
+
+            # Get the values for the bands from the PPSD
+            for idx, curper in enumerate(per):
+                if freslow[idxfre] < 1.0/curper < curfreUp:
+                    perGoodOBSPY.append(1.0/curper)
+                    pervalGoodOBSPY.append(perval[idx])
+
+            # Read in the results obtained from a Mustang flat file
+            file_dataIRIS = os.path.join(
+                PATH, 'IRISpdfExample')
+            fh = open(file_dataIRIS, 'r')
+
+            # We will now read in line by line the IRIS file and save
+            # a limited number of values
+            # Array of data values that will be within our frequency band
+            goodlines = []
+            # Distinct frequency values that we will want
+            freqdistinct = []
+            for line in fh:
```

Comment:
This loop can be replaced by `data = np.genfromtxt(file_dataIRIS, comments='#', separator=',')`.